src/actions/create.js

```diff
@@ -56,6 +56,12 @@ const askAppFolder = appId =>
     appFolder => validateAppFolder(appFolder),
   ])
 
+const askTypeScript = () =>
+  ask('Do you want to use TypeScript?', null, 'list', ['Yes', 'No']).then(
```

Comment:
Can we switch the answers ‘Yes’ and ‘No’, in order to make JS fixtures the default modus?

Also, perhaps should write the question a bit different? In theory TS can be used when in JS mode (i.e. it compiles TS code since the last version). This options presents the developer with a full TS boilerplate

Reply:
I think since the question is in the context of `lng create` the user will understand that it pertains to the generation of the boilerplate code. I'm not sure how to reword without it sounding a bit verbose:

- Do you want to use TypeScript in the boilerplate?
- Do you want the boilerplate to use TypeScript?